packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -252,7 +266,7 @@ export async function getAssetInfoAsync(asset: AssetRef): Promise<AssetInfo> {
 
   checkAssetIds([assetId]);
 
-  const assetInfo = await MediaLibrary.getAssetInfoAsync(assetId);
+  const assetInfo = await MediaLibrary.getAssetInfoAsync(assetId, options ?? {});
```

Comment:
What would you say if we put it as a default value of the argument?
```suggestion
  const assetInfo = await MediaLibrary.getAssetInfoAsync(assetId, options);
```

Reply:
This definitely looks more explicit. So yes 👍